src/app/Processor/ProcessorEdit/ConfigurationForm/CustomJsonSchemaBridge.tsx

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-unsafe-argument,@typescript-eslint/no-unsafe-return,@typescript-eslint/no-unsafe-member-access,@typescript-eslint/explicit-function-return-type */
+
+import React from "react";
+import { TFunction } from "react-i18next";
+import JSONSchemaBridge from "uniforms-bridge-json-schema";
+import { Popover } from "@patternfly/react-core";
+import { HelpIcon } from "@patternfly/react-icons";
+
+/**
+ * Returns an example string formatted (not localized) for the form or undefined if the field has no example text
+ * @param exampleText
+ * @returns
+ */
+
+const getExampleText = (exampleText: string) =>
+  typeof exampleText !== "undefined" ? `Example: ${exampleText}` : undefined;
+
+/**
+ * Returns a label tooltip element for the form or undefined if the field has no description
+ * @param name
+ * @param content
+ * @returns
+ */
+const getLabelIcon = (name: string, content: string) =>
+  typeof content !== "undefined" ? (
+    <Popover bodyContent={content}>
+      <button
+        type="button"
+        aria-label={`More info for ${name}`}
+        onClick={(e) => e.preventDefault()}
+        aria-describedby="form-group-label-info"
+        className="pf-c-form__group-label-help"
+      >
+        <HelpIcon noVerticalAlign />
+      </button>
+    </Popover>
+  ) : undefined;
+
+/**
+ * CustomJsonSchemaBridge generates the appropriate attributes for uniforms-patternfly
+ * based on the incoming model data
+ */
+export class CustomJsonSchemaBridge extends JSONSchemaBridge {
```

Comment:
Reusing this custom uniforms bridge implemented for connectors. Not sure we'll need everything here but let's wait until all the new actions/sources are added before proceeding with a cleanup.